### PR TITLE
Feature/130/ra dashboard/alicat

### PR DESCRIPTION
**Pull request by adowski**
Collaborated with @cathleenjia !!!! :) <3 xD rawr

## Style RA Dashboard

- Created `TenantCard` component
- Styled Referral Agency landing page to align with Figma design
- Made very tiny changes to `NavBar` and updated seeds.rb with a more realistic photo

### Related PRs

None

### Migrations

None

### Tests Performed, Edge Cases

None

### //TODO

- Add status tag helper (function that renders status tags) to utils.js so that we can use it everywhere (just wanted to get this PR in so Cat can use the Card component)
- remove inline styling and make classes

### Screenshots

image

CC: @wilsonw926

### CODE REVIEW

File: app/javascript/components/individual/TenantCard.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  }
+
+  redirectToTenantPage = () => {
+    window.location='/tenants/' + this.state.tenant.id;
+  }
+
+  render() {
+    return(
+      <div onClick={this.redirectToTenantPage} className="tenant-card">
+
```
Comment (wilsonw926) [inline app/javascript/components/individual/TenantCard.jsx]: remove this extra line here.
